2014/05/06 20:13:32 [error] [exception.CException] exception 'CException' with message 'SaludoController cannot find the requested view "index".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 C:\xampp\htdocs\yiiprimero\protected\controllers\SaludoController.php(8): CController->render('index')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SaludoController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('saludo')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yiiprimero/index.php?r=saludo
---
2014/05/06 20:24:41 [error] [exception.CException] exception 'CException' with message 'PruebaController cannot find the requested view "index".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php(8): CController->render('index')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PruebaController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba
---
2014/05/06 21:38:41 [error] [exception.CException] exception 'CException' with message 'Property "Prueba.id" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\yiiprimero\protected\views\prueba\index.php(14): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#6 C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php(11): CController->render('index', Array)
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PruebaController->actionIndex()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba
---
2014/05/06 21:44:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/view')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/view&id=1
HTTP_REFERER=http://localhost/yiiprimero/index.php?r=prueba
---
2014/05/06 21:47:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('view')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/view')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/view&id=1
HTTP_REFERER=http://localhost/yiiprimero/index.php?r=prueba
---
2014/05/07 17:28:57 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit
---
2014/05/07 17:29:07 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit/1')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit/1
---
2014/05/07 17:36:56 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit/1')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit/1
---
2014/05/07 17:36:57 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit/1')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit/1
---
2014/05/07 17:36:57 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit/1')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit/1
---
2014/05/07 17:37:20 [error] [exception.CException] exception 'CException' with message 'PruebaController and its behaviors do not have a method or closure named "endWitdget".' in C:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 [internal function]: CComponent->__call('endWitdget', Array)
#1 C:\xampp\htdocs\yiiprimero\protected\views\prueba\edit.php(17): PruebaController->endWitdget()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('edit', Array, true)
#6 C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php(29): CController->render('edit', Array)
#7 [internal function]: PruebaController->actionEdit('1')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PruebaController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PruebaController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit&id=1
---
2014/05/07 17:43:05 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit&ci=1
---
2014/05/07 17:44:20 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit&ci=1
---
2014/05/07 18:05:01 [error] [exception.CException] exception 'CException' with message 'Property "CRequiredValidator.messague" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(189): CComponent->__set('messague', 'Campos Requerid...')
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(287): CValidator::createValidator('required', Object(Prueba), 'nombre,descripc...', Array)
#2 C:\xampp\htdocs\yii\framework\base\CModel.php(260): CModel->createValidators()
#3 C:\xampp\htdocs\yii\framework\base\CModel.php(304): CModel->getValidators('usuario')
#4 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1414): CModel->isAttributeRequired('usuario')
#5 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(597): CHtml::activeLabelEx(Object(Prueba), 'usuario', Array)
#6 C:\xampp\htdocs\yiiprimero\protected\views\prueba\edit.php(5): CActiveForm->labelEx(Object(Prueba), 'usuario')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('edit', Array, true)
#11 C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php(30): CController->render('edit', Array)
#12 [internal function]: PruebaController->actionEdit('1')
#13 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PruebaController), Array)
#14 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PruebaController), Object(ReflectionMethod), Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('prueba/edit')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/yiiprimero/index.php?r=prueba/edit&id=1
---
2014/05/07 18:16:14 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:16:14 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:23:12 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:23:12 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:25:07 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:25:07 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:26:14 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:26:14 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:39:58 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:39:58 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:40:44 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:40:44 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (26)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:43:06 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (23)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:43:06 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (23)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:47:02 [warning] [application] Failed to set unsafe attribute "usuario" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (23)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/07 18:47:02 [warning] [application] Failed to set unsafe attribute "password" of "Prueba".
in C:\xampp\htdocs\yiiprimero\protected\controllers\PruebaController.php (23)
in C:\xampp\htdocs\yiiprimero\index.php (13)
2014/05/08 13:37:09 [error] [exception.CException] exception 'CException' with message 'La propiedad "CWebApplication"."themes" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('themes')
#1 C:\xampp\htdocs\yiiprimero\themes\classic\views\layouts\main.php(9): CModule->__get('themes')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\yiiprimero\themes\classic\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yiiprimero\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/yiiprimero/index.php
---
2014/05/08 13:48:31 [error] [exception.CException] exception 'CException' with message 'La propiedad "CWebApplication"."themes" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('themes')
#1 C:\xampp\htdocs\yiiprimero\themes\classic\views\layouts\main.php(9): CModule->__get('themes')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\yiiprimero\themes\classic\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\yiiprimero\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\yiiprimero\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/yiiprimero/index.php
---
